src/Adapter/Http.php

```diff
@@ -479,7 +479,7 @@ protected function _basicAuth($header)
 
         // Decode the Authorization header
         $auth = substr($header, strlen('Basic '));
-        if ($auth === false) {
+        if ($auth === '') {
```

Comment:
this change was highlight by psalm

see change log for substr

```

8.0.0 | The function returns an empty string where it previously returned false.

```

Reply:
@Ocramius what do you think ?

Reply:
Was a test failing here?

Reply:
Yeah, we should make sure this path is hit with a test 🤔